dataAccessObjects.py

```diff
@@ -61,6 +61,13 @@ def checkUserExisted(self,userName):
             + 'WHERE user_name="{userName}"'.format(userName=userName)
         return self.queryOneValue(sql)
 
+    def createNewUser(self,userName,userPassword):
+        cursor = self.db.cursor
+        sql = 'INSERT INTO user (user_name,user_password) VALUES (%s,%s)'
+        data = (userName,userPassword)
+        ret = cursor.execute(sql,data)
```

Comment:
Alright, let us modify it in the future.
Now just extract it to be a Dao method